### ISSUE TITLE
Extension broken in Firefox 50: Needs signing

### ISSUE THREAD

**Issue by benjaoming** [codecov/browser-extension#39]
Thanks for the great plugin!!

I'm sorry to see that things aren't being approved speedily enough on the Mozilla Add-On site.

It's even worse now that Firefox seems to have made the signing of XPI files mandatory.

![screenshot from 2016-11-20 09-58-26](https://cloud.githubusercontent.com/assets/374612/20461756/eea3278e-af07-11e6-852c-a1d24f07bee7.png)

**Comment by benjaoming** [comment]
p.s. but requiring a signature on the XPI file is a security improvement, seems okay :)

**Comment by benjaoming** [comment]
More info: https://developer.mozilla.org/en-US/Add-ons/Distribution

**Comment by nicklan** [comment]
Any update on this?  It's now impossible to install the plugin if you're on a recent version of firefox.  The one at the add-ons site isn't compatible, and the newer xpi isn't signed, and signing is enforced with no override after firefox 48.

It's been almost a year, since this report, does it really take that long to get a new version signed?

**Comment by benjaoming** [comment]
@nicklan Firefox has dumped extensions and will only support WebExtensions in the future: https://blog.mozilla.org/addons/2017/08/10/upcoming-changes-compatibility/

**Comment by nicklan** [comment]
@benjaoming yeah, i'm aware of that.  i didn't realize that the current "newer" thing linked wasn't a webextention.  what's the plan for the plugin going forward then? should there be a new issue for converting to a firefox webextention?  should this issue be closed.

thanks for the reply!

**Comment by benjaoming** [comment]
See #44 :)

I don't think the issue should close, but perhaps rather the whole repo?
